ocs_ci/deployment/helpers/hypershift_base.py

```diff
@@ -172,6 +193,19 @@ def get_hosted_cluster_type(cluster_name=None):
     return ocp_hosted_cluster_obj.get()["spec"]["platform"]["type"].lower()
 
 
+def get_current_nodepool_size(name):
+    """
+    Get existing nodepool of HyperShift hosted cluster
+    Args:
+        name (str): name of the cluster
+    Returns:
+         str: number of nodes in the nodepool
+    """
+    logger.info(f"Getting existing nodepool of HyperShift hosted cluster {name}")
+    cmd = f"oc get --namespace clusters nodepools | awk '$1==\"{name}\" {{print $4}}'"
```

Comment:
Generally I would prefer to parse the json instead of the plain output as it will be more robust, but I know it is not possible in all the cases (not sure about this particular case). So take it more like suggestion for the future.

Reply:
I will address it later if you allow. it is old method it was just refactored to become a function, for easier access.